src/js/index.js

```diff
@@ -35,7 +35,7 @@ store.subscribe(throttle(model.update, 16));
 var ui = require('./components');
 
 // name model Scene
-model.Scene.name = 'Scene';
+model.Scene._name = 'Scene';
```

Comment:
I'm +1 to explicit getters and setters... this though I worry about b/c the way we construct a scene is a little different in my PR, why do we not just use .name like with the other primitives?